package.json

```diff
@@ -39,6 +39,7 @@
     "docker:up": "docker-compose -f docker-compose-wsl2.yml down && docker-compose -f docker-compose-wsl2.yml up --build --scale chrome=4",
     "eslint": "npm run precommit",
     "lint-staged": "lint-staged",
+    "pack-all": "npx lerna exec --scope=botframework* -- npm pack --pack-destination=$PWD && zip tarball.zip *.tgz",
```

Comment:
Not sure if this is critical: `$PWD` won't resolve on windows shells

Reply:
It's not critical. 😉 If someone want to use this script, I would kindly ask them to use GitHub Codespaces. 🤣